pkg/kubelet/kubelet_pods_test.go

```diff
@@ -1305,6 +1305,57 @@ func TestMakeEnvironmentVariables(t *testing.T) {
 				},
 			},
 		},
+		{
+			name:               "test_duplicated_keys",
+			ns:                 "test",
+			enableServiceLinks: &falseValue,
+			container: &v1.Container{
+				EnvFrom: []v1.EnvFromSource{
+					{
+						ConfigMapRef: &v1.ConfigMapEnvSource{LocalObjectReference: v1.LocalObjectReference{Name: "test-config-map"}},
```

Comment:
Is it an issue that this is `test-config-map` but the name of the config map object is `test-configmap`?

Reply:
Ops, I will fix it.

Reply:
fixed.
8532133